db/schema.rb

```diff
@@ -23,13 +23,17 @@
     t.string   "state"
     t.string   "city"
     t.string   "email"
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
+    t.datetime "created_at",                                   null: false
+    t.datetime "updated_at",                                   null: false
     t.boolean  "hiring"
-    t.string   "status",       default: "uncontacted"
+    t.string   "status",               default: "uncontacted"
     t.float    "latitude"
     t.float    "longitude"
-    t.integer  "hire_count",   default: 0
+    t.integer  "hire_count",           default: 0
+    t.string   "zip_code"
```

Comment:
I wasn't sure if I should make this a string or an integer... I chose string... but now that I look at it, I think it should probably be an integer so we can validate that its an actual zip code with no letters... thoughts?